aemcmc/conjugates.py

```diff
@@ -1,12 +1,10 @@
 import aesara.tensor as at
+from aeppl.opt import NoCallbackEquilibriumDB
```

Comment:
If `NoCallbackEquilibriumDB` were to change in `aeppl` would we necessarily want this change to be propagated in `aemcmc`? Wondering if it make more sense to subclass `EquilibriumDB` directly in `opt.py` here.